.github/workflows/integration.yml

```diff
@@ -183,6 +183,53 @@ jobs:
         run: |
           docker compose down
 
+  flightsql-meson:
```

Comment:
Added this here for completeness, though not sure if we feel like it is overkill

The flightsql/snowflake tests additionally test the Python libraries, which I did not add since we are not using meson for the Python packages

Reply:
IMO it's a bit much - is there a reason to think the build system would affect these tests?

Reply:
No expectations that the test results would be different - was just using this as validation that the Meson configuration properly sets options

Instead of separate jobs I just created one job where all options are being set. If that's too much still we can scale it back to just SQLite